### PR TITLE
Listing media galery

### PR DESCRIPTION
**Pull request by Karan3036**
Created the Listing Media Gallery

### CODE REVIEW

File: force-app/main/default/lwc/checkListStatus/checkListStatus.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<template>
+    <div class="container">
+        <div class="header">
+            <h1>Checklist Progress</h1>
+            <div class="buttons">
+                <button class="edit-btn" onclick={handleShowEditModal}>
+                    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (MRG-MV) [inline force-app/main/default/lwc/checkListStatus/checkListStatus.html:7]: Remove SVG and Path and use Static Resource if possible

File: force-app/main/default/lwc/editCheckListModal/editCheckListModal.js [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+                Name: '',
+                MVEX__Field_Name__c: '',
+                MVEX__Operator__c: '',
+                MVEX__Value__c: ''
+            };
+            this.checklistItems = [...this.checklistItems, newItem];
+    
+            // Delay the scroll to ensure the DOM has updated
+            // If we remove the setTimeout, the scroll will not work
+            setTimeout(() => {
```
Comment (MRG-MV) [inline force-app/main/default/lwc/editCheckListModal/editCheckListModal.js:175]: Timeout is set as 0 so check if it can work without it or not

**Review by MRG-MV — CHANGES_REQUESTED** [review]

File: force-app/main/default/lwc/checkListStatus/checkListStatus.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    @track checklistItems = []; // Initialize as an empty array
+    @track showEditModal = false;
+    @api objectName;
+    @api recordId;
+    @track isSpinner = true;
+
+    @wire(CurrentPageReference)
+    setCurrentPageReference(pageRef) {
+        console.log('Page Reference:', JSON.stringify(pageRef, null, 2)); // Debugging line
+        this.objectName = pageRef.attributes.objectApiName;
```
[the remark this answers is not in the document]
Comment (Karan3036) [reply force-app/main/default/lwc/checkListStatus/checkListStatus.js:17]: I have changed the decorators for objectName and recorded.

File: force-app/main/default/lwc/editCheckListModal/editCheckListModal.js (hunk 2 above)
Comment (Karan3036) [reply force-app/main/default/lwc/editCheckListModal/editCheckListModal.js:175]: If I remove the setTimeout, the scroll bar will not work. That's why I have added the setTimeout.

Comment (Karan3036) [reply force-app/main/default/lwc/editCheckListModal/editCheckListModal.js]: Removed the setTimeout out and put the logic into rendered callback.

**Review by MRG-MV — APPROVED** [review]
Approve

**Pull request merged**